src/routes/safe/components/Apps/index.jsx

```diff
@@ -0,0 +1,140 @@
+// @flow
+import React, { useState, useEffect, useCallback } from 'react'
+import styled from 'styled-components'
+
+import sendTransactions from './sendTransactions'
+
+const AppsWrapper = styled.div`
+  margin: 15px;
+`
+
+const operations = {
+  SEND_TRANSACTIONS: 'sendTransactions',
```

Comment:
This is a tentative approach of the messages to communicate between Safe and the third-party apps.